services/util/CMakeLists.txt

```diff
@@ -1,12 +1,9 @@
 add_library(services.util ${EMIL_EXCLUDE_FROM_ALL} STATIC)
 
-protocol_buffer_echo_cpp(services.util MessageCommunicationSecurity.proto)
```

Comment:
Only MessageCommunicationSecured depends on MbedTLS. Without MbedTLS, having Echo is still possible and the most common usecase. I'd like to see only EchoOnMessageCommunicationSymmetricKey and MessageCommunicationSecured to be moved inside the if (EMIL_INCLUDE_MBEDTLS)

Reply:
Okay, fixed